crates/amaru/src/consensus/store.rs

```diff
@@ -0,0 +1,31 @@
+use super::header::Header;
+use miette::Diagnostic;
+use pallas_crypto::hash::Hash;
+use std::fmt::Display;
+use thiserror::Error;
+
+pub mod rocksdb;
+
+#[derive(Error, Diagnostic, Debug)]
```

Comment:
We probably want to have a common strategy for defining errors, e.g. use `thiserror`